gno.land/cmd/gnofaucet/main.go

```diff
@@ -23,7 +23,7 @@ func main() {
 	)
 
 	if err := cmd.ParseAndRun(context.Background(), os.Args[1:]); err != nil {
-		_, _ = fmt.Fprintf(os.Stderr, "%+v", err)
+		_, _ = fmt.Fprintf(os.Stderr, "%+v\n", err)
```

Comment:
Why not? Since we are already ignoring return values. Looks cleaner.
```suggestion
                fmt.Fprintf(os.Stderr, "%+v\n", err)
```

Reply:
I can't find any linter setup that complains about this.

IMO, removing unnecessary assignments to `_` makes the code cleaner.

I have noticed in some places we are using `fmt.Fprintf(...)` and in some `_, _ = fmt.Fprintf(...)`
Do we have any criteria for when to use `_` to purposefully ignore errors and when not?